ci/ansible/roles/pulp-certs/tasks/qpid.yml

```diff
@@ -0,0 +1,247 @@
+---
+- name: create temporary working directory
+  tempfile:
+    state: directory
+  register: temp_dir
+
+- set_fact:
+    # when passing `-i root@hostname,` removes the `root@`
+    hostname: "{{ inventory_hostname.rpartition('@')[-1] }}"
+    tempdir: "{{ temp_dir.path }}"
+
+- set_fact:
+    instdir: "/etc/pki/qpid"
+    clientdir: "{{ tempdir }}/client"
+    pwdfile: "{{ tempdir }}/password"
+    seedfile: "{{ tempdir }}/seed"
+    valid: 12
+    ca_path: "/etc/pki/pulp/ca.crt"
+    ca_key_path: "/etc/pki/pulp/ca.key"
+    ca_name: "ca"
+    db_password: "pulp"
+    trust: "TCu,Cu,Tuw"
+    client_subject: "CN=client,O=pulp,ST=Alabama,C=US"
+    client_nickname: "client"
+    broker_subject: "CN={{ hostname }},O=pulp,ST=Alabama,C=US"
+    broker_nickname: "broker"
+
+# SETUP BASE NSS db and Certs
+
+- name: Create the password file
+  lineinfile:
+    dest: "{{ pwdfile }}"
+    line: "{{ db_password }}"
+    create: yes
+
+- name: Create the main NSS db
+  command: certutil -N -d {{ tempdir }} -f {{ pwdfile }}
```

Comment:
I think this is going to require `--empty-password` in FIPS mode as well. See: https://bugzilla.redhat.com/show_bug.cgi?id=1401606

Reply:
@daviddavis code updated, I used the empty-password and tested in f26, 27, rhel7.5 and also rhel7.5 with FIPS enabled.